site/blog/2022-10-27-clean-your-aws-account.mdx

```diff
@@ -0,0 +1,36 @@
+---
+slug: clean-your-aws-account
+title: Clean your AWS account just by clicking a button
+authors: [yrobla]
+---
+
+# Clean your AWS account by simply clicking a button
+
+One of the main complexities of working with AWS is the amount of resources that can be created, either by manual or automated workflows. The chained resources and their dependencies can quickly get out of control,
+and it could be a nightmare to clean up then.
+
+This is a common problem when using tests accounts, either for developing or testing some integrations. Often these accounts have resources out of control, causing you to incur on extra costs, or causing unexpected conflicts.
+Cleaning those manually can be a hard task... manually visiting each resource, investigating the dependencies, manually cleaning... it is a time-consuming, error-prone task. If we consider that those resources may be created in
+multiple regions, the manual procedure can become a very difficult task.
+
+To solve this problem you can use our IaSQL engine. By simply clicking a button, our engine will get an snapshot of the status of your cloud, and will schedule deletion of the existing resources and its dependencies
+until your account is totally cleaned.
+
+_Please note that our engine can offer this cleanup only for the resources covered by our system. You can get a full coverage list <a href="https://github.com/iasql/iasql-engine#aws-services-with-significant-api-coverage">here</a>._
```

Comment:
Not directly related with this blogpost, but before launching this we should update this section https://github.com/iasql/iasql-engine#aws-services-with-significant-api-coverage